theano/gpuarray/subtensor.py

```diff
@@ -733,6 +736,7 @@ def c_code(self, node, name, inputs, outputs, sub):
         if (!%(inplace)s) {
           %(out)s = theano_try_copy(%(out)s, %(x)s);
           if (%(out)s == NULL)
+            PyErr_SetString(PyExc_MemoryError, "Could not allocate copy of input.");
```

Comment:
Oh, I'm missing curly braces there, which would explain why things get weird.
I'll fix that later.